tests/monday_type_tests.py

```diff
@@ -84,3 +84,77 @@ def test_should_succeed_when_to_primitive_returns_str_when_passed_in_an_int_floa
     eq_(int_value,'1')
     eq_(float_value,'1.0')
     eq_(str_value,'text')
+
+
+
+def test_should_succeed_when_to_native_returns_a_bool_when_passing_a_checkboxvalue_value_with_api_data():
```

Comment:
In the future, please indicate what type you are testing in the method name to avoid merge confusion in the future.  Something like def test_checkbox_type_should_suceed_when...

Reply:
Ok